### DISCUSSION TITLE
GraphQL interpolation between query sections.

### DISCUSSION THREAD

**Discussion by 3goats** [hasura/graphql-engine#6618] [General]
Apologies for raising this question here, but I'm new to GraphQL/Hasura and don't know if I'm barking up the wrong tree.

I have a graphQL query that essentially queries two datasources. I need to pass some data returned from the first part of the query to the second part. i.e. Pass the returned `hostname` from the `config` query to the `CommonName` in the `GetCertificate` query. e.g.  

```
query get_config {
  config {
    id
    hostname
  }
  getCertificate(
      CommonName:  $hostname) {
    Certificate
    Key
  }
  }
```

So basically perform a single query but pass in some of the output from the first part of the query to the second part. 

I have also asked the question here on Stack-overflow however, I'm not sure if its possible to implement within Hasura. 

https://stackoverflow.com/questions/66350642/graphql-interpolation-between-queries

**Comment by tirumaraiselvan** [comment]
What are the two data sources? i.e. where is `config` retrieved from and where is `getCertificate`? 

If `getCertificate` is from a remote schema, then a remote schema relationship would be the right way to go: https://hasura.io/docs/latest/graphql/core/databases/postgres/schema/remote-relationships/remote-schema-relationships.html#remote-schema-relationships
